src/lib/ecore/ecore_time.c

```diff
@@ -55,14 +55,30 @@ ecore_time_get(void)
 EAPI double
 ecore_time_unix_get(void)
 {
-//#ifdef HAVE_GETTIMEOFDAY
+
+
+   //struct timeval {
+     //          time_t tv_sec; / * segundos * /
+       //        suseconds_t tv_usec; / * microssegundos * /
+         //  };
+
+#ifdef HAVE_GETTIMEOFDAY
+
    struct timeval timev;
 
    gettimeofday(&timev, NULL);
    return (double)timev.tv_sec + (((double)timev.tv_usec) / 1000000);
-//#else
-//# error "Your platform isn't supported yet"
-//#endif
+   
+#else
```

Comment:
Maybe we should guarantee that only windows systems enter here if an `#ifdef _WIN32`

Reply:
Now that gettimeofday is on evil, this file should be restored to `devs/joaoantoniocardoso/native-rebased/ecore`.